webapp/src/linterChecks/Arsh.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
You're working on the same branch as before and I used squash&merge so git doesn't know this file is already in `winter2022/main`. Can you either rebase or update your branch? You should also be making new branches for new features, just one of those weird new workflows to think about when using git :)
If you rebase make a new branch, but if you merge `winter2022/main` into this one just keep using the same one and we'll do a walkthrough for your next feature!

Also see: Slack and this comment I made on Signe's branch for more info (and to save me typing lol) https://github.com/NorimaConsulting/training-and-onboarding/pull/5#discussion_r786217733